src/app.js

```diff
@@ -302,6 +302,8 @@ app.get('/index', async (req, res) => {
         // check if index exists
         let createdIndex = await create_index_if_not_exists(db, 'id', collection_name);
         createdIndex = await create_index_if_not_exists(db, 'meta.lastUpdated', collection_name) || createdIndex;
+        createdIndex = await create_index_if_not_exists(db, 'meta.source', collection_name) || createdIndex;
+        createdIndex = await create_index_if_not_exists(db, 'meta.security', collection_name) || createdIndex;
```

Comment:
since `meta.security` is an array of structs, would this just work like a charm?
`meta.source` is a string so it should be good, but not sure if this would work with a struct type data??
if it does, then awesome!

Reply:
added multi key index to handle this